projects/fmcomms2/common/fmcomms2_bd.tcl

```diff
@@ -32,7 +32,7 @@ ad_ip_parameter axi_ad9361 CONFIG.ID 0
 # set to 1 for CORDIC or 2 for POLYNOMIAL
 ad_ip_parameter axi_ad9361 CONFIG.DAC_DDS_TYPE 1
 ad_ip_parameter axi_ad9361 CONFIG.DAC_DDS_CORDIC_DW 14
-ad_connect sys_200m_clk axi_ad9361/delay_clk
```

Comment:
Who is going to provide this new clock in case of the other carriers ?

Reply:
good question :))

Reply:
I think it's time to find a better name for this clock, which does not contains its frequency.